tests/integration/v2/slashing/slashing_test.go

```
@@ -0,0 +1,176 @@
+	_ "cosmossdk.io/x/bank"     // import as blank for app wiring
+	bankkeeper "cosmossdk.io/x/bank/keeper"
+	banktestutil "cosmossdk.io/x/bank/testutil"
+	_ "cosmossdk.io/x/consensus"    // import as blank for app wiring
+	_ "cosmossdk.io/x/distribution" // import as blank for app wiring
+	distrkeeper "cosmossdk.io/x/distribution/keeper"
+	_ "cosmossdk.io/x/mint"         // import as blank for app wiring
+	_ "cosmossdk.io/x/protocolpool" // import as blank for app wiring
+	_ "cosmossdk.io/x/slashing"     // import as blank for app wiring
+	slashingkeeper "cosmossdk.io/x/slashing/keeper"
+	"cosmossdk.io/x/slashing/testutil"
+	"cosmossdk.io/x/slashing/types"
```

golangci-lint (1.62.2)

ST1019: package "cosmossdk.io/x/slashing/types" is being imported more than once

(stylecheck)